src/settings.yml

```diff
@@ -3,11 +3,13 @@ strategy: polling
 no_screen_padding: 'no'
 dark_mode: 'no'
 static_data: ''
-polling_verb: get
-polling_url: https://api.tender.wtf/hc/stats?bearer={{ bearerToken }}
-polling_headers: ''
-polling_body: ''
-id: 63345
+polling_verb: post
+polling_url: |
+  https://api.hardcover.app/v1/graphql
+  https://www.randomnumberapi.com/api/v1.0/randomnumber
```

Comment:
would prefer to use https://usetrmnl.com/custom_plugin_example_data.json  here if/when they accept POST requests